extensions/todo-list/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Todo List Changelog
 
+## Fix Handling URLs Without a Scheme (2022-03-17)
```

Comment:
```suggestion
## [Fix Handling URLs Without a Scheme] - 2022-03-17
```

Reply:
We added strict format of the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) - could you address this?